apps/www/components/Flyer/index.tsx

```diff
@@ -29,6 +31,37 @@ export type MetaProps = {
   [x: string]: unknown
 }
 
+const RenderValue: React.FC<{
+  value: CustomDescendant[]
+}> = ({ value }) => (
+  <SlateRender
+    value={value}
+    schema={flyerSchema}
+    raw
+    skip={['flyerOpeningP']}
+  />
+)
+
+const RenderWithPaynote: React.FC<{
+  value: CustomDescendant[]
+  tileId?: string
+  repoId: string
+  documentId: string
+}> = ({ value, tileId, repoId, documentId }) => {
+  const seed = useMemo(() => Math.random(), [])
+  let idx = 2
```

Comment:
Why is this default-value `2`? A comment could be helpful for the readability 👀

Reply:
bhahahaha vade retro magic number!!!